README.md

```diff
@@ -41,6 +41,32 @@ vsce package # `npm i -g vsce` if not already installed
 Install locally:
 cmd+shift+p > "Extensions: Install from VSIX"
 
+<br />
+
+---
+
+Alternative:
```

Comment:
Let's combine this alternative with my OG instructions, they are quite similar

Reply:
Yes. Could you please combine them?
I don't know how you'd like your part integrated with mine